### PR TITLE
chore: add CODEOWNERS

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Add @smoya @jonaslagoni @BOLT04 and me as CODEOWNERS of this project:

- @smoya because he is a expert in the backend and actively participated in the discussion about architecture of the API
- @jonaslagoni like Sergio
- @BOLT04 like Sergio and Jonas, but also contributed to other parts of our tools, so I think that it's a good person. 
- me because I started that project and wrote first path - https://github.com/asyncapi/server-api/pull/7 

@BOLT04 If you think you don't want to be a part of this project let us know, even if we merge this PR and you don't respond. We can always add/remove you later, no rush :)

### CODE REVIEW

**Comment by jonaslagoni** [comment]
Unless it is required for some reason, I dont think it makes sense to add me as codeowner.

Especially if you have @smoya and @BOLT04 to join you 🙂

File: CODEOWNERS
Comment (smoya) [inline CODEOWNERS:9]: I tried to bribe @jonaslagoni with some cinema tickets but didn't work, even though [The Matrix Resurrections](https://www.imdb.com/title/tt10838180/) is almost here... So I think we should remove it from the list.
```suggestion
* @smoya @BOLT04 @magicmatatjahu @github-actions[bot]
```

**Comment by smoya** [comment]
> @BOLT04 If you think you don't want to be a part of this project let us know, even if we merge this PR and you don't respond. We can always add/remove you later, no rush :)

I would not merge this PR if @BOLT04 does not approve it. I would wait for its response or rather remove it now and add it later if they wish.

**Review by BOLT04 — APPROVED** [review]
Hi @magicmatatjahu, thank you so much for the opportunity and the tag 😃.
I'll do my best to contribute and collaborate with you all on this project (and on the whole community 😄)

**Comment by magicmatatjahu** [comment]
@BOLT04 Awesome to hear that!, thanks I don't know if you read my message to you on Slack, so I will paste it here because it is very important. If you still want be a maintainer, please agree or not :)

> Hello! I opened PR with adding codeowners for server-api  and also I added you. If you have objections, please write them in the PR's comment https://github.com/asyncapi/server-api/pull/18 Of course with benefits (like membership in the TSC - more info here https://github.com/asyncapi/community/blob/master/CHARTER.md) you will have also responsibilities. I chose you because you participated with several projects in our org like parser-js, server-api and also github-action, so I would like to have you as a maintaner in our project. Of course no rush, you don't have to agree, just let us know if you accept these conditions or not :)

**Comment by BOLT04** [comment]
@magicmatatjahu thanks for the notice 👍, I've sent some questions to you on slack regarding timezones and voting 🙂

**Comment by magicmatatjahu** [comment]
@BOLT04 Hello! Could you add yourself as TSC Member like in this PR https://github.com/asyncapi/community/pull/179 in community repo? Please add info about your company and add yourself to the `server-api` projekt as maintainer :) We can do it for you, but it will be better that you will do, ok? :)

**Pull request merged**
